packages/account-api/src/api/provider.ts

```diff
@@ -1,9 +1,19 @@
 import type { EntropySourceId, KeyringAccount } from '@metamask/keyring-api';
 
+export enum AccountProviderType {
```

Comment:
Would this limit the addition of new providers in a dynamic way? e.g., the development of a tron provider that momentary lives in the client